src/Mobs/Monster.cpp

```diff
@@ -1129,7 +1135,6 @@ bool cMonster::WouldBurnAt(Vector3d a_Location, cChunk & a_Chunk)
 		(a_Chunk.GetSkyLight(RelX, RelY, RelZ) == 15) &&             // In the daylight
```

Comment:
Macros everywhere! The main thing would be to remember to highlight to contributors that it needs to be done if they are using the chunk parameter. I suppose cMonster::Tick could call GET_AND_VERIFY... before it does work and after to account for all situations, but for some mobs the extra processing is needless since they don't override.